### PR TITLE
Fix UI/UX for creating course modules, lessons, quizzes

### LINKED ISSUE #613: Fix UI/UX for Courses Management Page
- Figure out a better UI/UX for course management

### LINKED ISSUE #675: Minor issue on the Student UI
Student dashboard shows the active courses.
But if the student enrols then the same course is seen twice

### PR DESCRIPTION
**Pull request by adityacp**
This PR closes #613, closes #675 

- Allow moderators to create, view and edit modules, lessons, quizzes from the manage course page.
- Removed All quizzes, All lessons, All modules options from the courses page.
- Show distinct courses in student dashboard

### CODE REVIEW

**Comment by adityacp** [comment]
courses
manage_course

Adding screenshots for UI changes.

**Comment by CruiseDevice** [comment]
LGTM

**Comment by prathamesh920** [comment]
If we create learning units (lessons and quizzes) then all get the order as 1.
So the student ```Next``` navigation will not work.
It should be incremental

File: yaksh/views.py [hunk 1]
```diff
@@ -2591,14 +2618,14 @@ def design_module(request, module_id, course_id=None):
                 for order, value in enumerate(add_values, start_val):
                     learning_id, type = value.split(":")
                     if type == "quiz":
-                        learning_unit = LearningUnit.objects.create(
+                        unit, status = LearningUnit.objects.get_or_create(
                             order=order, quiz_id=learning_id,
                             type=type)
                     else:
-                        learning_unit = LearningUnit.objects.create(
+                        unit = LearningUnit.objects.get_or_create(
```
Comment (prathamesh920) [inline yaksh/views.py]: ```unit, status``` should come  as returns tuple
and unit you need to pass to ```...learning_unit.add(*units_list)```, will fail if tuple is passed.

**Pull request merged**